data/yaksplained.json

```diff
@@ -0,0 +1,378 @@
+    "image": "https://cdn.discordapp.com/emojis/879749569964015646.png",
+    "remark": "How did this grow so fast?",
+    "name": "Hairyak"
+  },
+  {
+    "use_in_serenity": "U+10CD01",
+    "creation_history": "Created by Agni on July 17th, 2021. Reason: [requested by Andreas](https://discord.com/channels/830522505605283862/830525235803586570/865881764111450112). Also relevant: [the associated meme](https://discord.com/channels/830522505605283862/830525235803586570/867538791962378240).",
+    "description": "Shave incoming. Used for when you're about to solve/currently solving a problem. Or if you noticed a problem and want to solve it right now.",
+    "image": "https://cdn.discordapp.com/emojis/866030445388365884.png",
+    "remark": "This yak's about to be shaved.",
+    "name": "Imminentyakshave"
+  },
+  {
+    "use_in_serenity": "U+10CD15",
+    "creation_history": "Created by Agni on June 6th, 2021. Reason: symbols for the bot's LibJS test262 runner status. Here, it is used for all the tests that caused a runner exception. Starting with [this message](https://discord.com/channels/830522505605283862/830525235803586570/851168937131638784), and with encouragement by networkException a conversation commenced that led to the creation of custom emojis for all the test runner results. _These emojis will be referred to here as the neoyak group._",
+    "description": "Hacker terminal or coding-themed yak, for all your hacker needs. Used in conjunction with Terminal or pop culture hacker-related things. Also used as a ninja yak when something ninja-related comes up.",
+    "image": "https://cdn.discordapp.com/emojis/851188922512048199.png",
+    "remark": "Hax0r",
+    "name": "Neoyak"
+  },
+  {
+    "use_in_serenity": "U+10CD60",
+    "creation_history": "Created by Andreas on October 21st, 2021. Reason: [celebration of the :yakking: emoji](https://discord.com/channels/830522505605283862/830522505605283866/900713226810064967) (see below).",
+    "description": "This yak is having a party, compare to the default (i.e. Unicode) 🥳 emoji.",
+    "image": "https://cdn.discordapp.com/emojis/901168038248874045.png",
+    "remark": "Congratulations!",
+    "name": "Partyak"
+  },
+  {
+    "use_in_serenity": "U+10CD08",
+    "creation_history": "Created by Linus on October 9th, 2021. Reason: After discussing the :goldyak: origins (see above) with [Linus' blacked-out screenshot](https://discord.com/channels/830522505605283862/830525235803586570/896476288167071824), Sam asked [\":reyakted: emote wen?\"](https://discord.com/channels/830522505605283862/830525235803586570/896484164059156490) and Linus promptly created it. ",
+    "description": "(Note to the RFC editor: Insert an SCP joke here.)",
+    "image": "https://cdn.discordapp.com/emojis/896486220106956810.png",
+    "remark": "This yak is named \\[REDACTED\\], it was born on ■■■■■■■.",
```

Comment:
I assume the backslashes still work in Markdown

Reply:
sure, give it a minute and I'll show the generated stuff.